count_word.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+
+import ("fmt"
+	"log"
+	"io/ioutil"
+	"os"
+	"bufio"
+	"sync"
+	"strings"
+	)
+
+
+func reader(filename string) []string {
+    file, err := os.Open(filename)
+    if err != nil {
+           log.Fatal(err)
+    }
+    scanner := bufio.NewScanner(file)
+    scanner.Split(bufio.ScanLines)
+    var txtlines []string
+
+    for scanner.Scan() {
+        txtlines = append(txtlines, scanner.Text())
+    }
+    defer file.Close()
+    return txtlines
+}
+
+func counter(contents []string) map[string]int {
+    map_m := make(map[string]int)
+    for _, eachline := range contents {
+	str := strings.Replace(eachline, ".", "", -1)
+	str = strings.Replace(eachline, ",", "", -1)
+	for _,char := range strings.Split(str, " ") {
```

Comment:
should use [`strings.Fields`](http://golang.org/pkg/strings#Fields), it also removes trailing spaces.